### PR TITLE
NoUnneededControlParenthesesFixer -  add yield from

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by SpacePossum** [comment]
not sure I understand the failure

File: src/RuleSet.php [hunk 1]
```diff
@@ -286,13 +295,6 @@ final class RuleSet implements RuleSetInterface
             '@Symfony:risky' => true,
             'comment_to_phpdoc' => true,
             'final_internal_class' => true,
-            'function_to_constant' => ['functions' => [
```
Comment (julienfalque) [inline src/RuleSet.php]: This removal should be backported to 2.15.

Comment (SpacePossum) [reply src/RuleSet.php]: I think we should only update the rule sets on the next minor?

Comment (julienfalque) [reply src/RuleSet.php]: This is kind of a fix of the ruleset, it can be released in a patch version.

**Pull request merged**